### PR TITLE
COZMO-6231-Added docstrings for the robot properties that were missing them

### PR DESCRIPTION
**Pull request by MarkWesleyAtAnki**
COZMO-6231 Added docstrings for the robot properties that were missing them

### CODE REVIEW

File: src/cozmo/robot.py [hunk 1]
```diff
@@ -483,14 +484,23 @@ def __init__(self, conn, robot_id, is_primary, **kw):
 
         self._action_dispatcher = self._action_dispatcher_factory(self)
 
+        #: :class:`cozmo.util.Speed` Speed of the left wheel
```
Comment (gwatts) [inline src/cozmo/robot.py]: The type should have a colon after it .. i.e... ``...util.Speed`: Speed of...``

File: src/cozmo/robot.py [hunk 2]
```diff
@@ -483,14 +484,23 @@ def __init__(self, conn, robot_id, is_primary, **kw):
...
         self.left_wheel_speed = None
+        #: :class:`cozmo.util.Speed` Speed of the right wheel
         self.right_wheel_speed = None
-        self.light_height = None
+        #: :class:`cozmo.util.Distance` height of the lift from the ground (in :const:`MIN_LIFT_HEIGHT_MM` to :const:`MAX_LIFT_HEIGHT_MM` range)
```
Comment (gwatts) [inline src/cozmo/robot.py]: you can wrap these comments by also beginning subsequent lines with `#:`

File: src/cozmo/robot.py [hunk 3]
```diff
@@ -483,14 +484,23 @@ def __init__(self, conn, robot_id, is_primary, **kw):
...
+        self.lift_height = None
+        #: float: the current battery_voltage (not linear, but < 3.5 is low)
```
Comment (gwatts) [inline src/cozmo/robot.py]: can just be "battery voltage" without the underscore, no?

File: src/cozmo/robot.py [hunk 4]
```diff
@@ -473,6 +473,7 @@ def __init__(self, conn, robot_id, is_primary, **kw):
 
         self._is_ready = False
         self._pose = None
+        #: bool: is this the primary robot (always True currently)
```
Comment (msintov) [inline src/cozmo/robot.py]: Could reword this to say: "Specifies that this is the primary robot"

File: src/cozmo/robot.py [hunk 5]
```diff
@@ -483,14 +484,23 @@ def __init__(self, conn, robot_id, is_primary, **kw):
...
+        self.lift_height = None
+        #: float: the current battery_voltage (not linear, but < 3.5 is low)
         self.battery_voltage = None
+        #: int: the ID of the object currently being carried (-1 if none)
         self.carrying_object_id = -1
+        #: int: the ID of the object on top of the object currently being carried (-1 if none)
         self.carrying_object_on_top_id = -1
+        #: int: the ID of the object the head is tracking to (-1 if none)
         self.head_tracking_object_id  = -1
+        #: int: the ID of the object that robot is localized to (-1 if none)
```
Comment (msintov) [inline src/cozmo/robot.py]: Could reword this to "that the robot is..."

File: src/cozmo/robot.py (hunk 2 above)
Comment (msintov) [inline src/cozmo/robot.py]: Each of these descriptions should start with a capital letter to match the existing descriptions.

File: src/cozmo/robot.py [hunk 6]
```diff
@@ -483,14 +484,23 @@ def __init__(self, conn, robot_id, is_primary, **kw):
...
         self.localized_to_object_id = -1
+        #: int: the timestamp for the last image seen (>= 0)
```
Comment (gwatts) [inline src/cozmo/robot.py]: This timestamp is another millisecond timestamp relative to the boot time of the robot, correct?  along the lines of `last_observed_robot_timestamp` and `last_tapped_robot_timestamp` in `object.py`

We should probably clarify the docs for those types to stay how they're generated.. would be nice if this one was also called `last_image_robot_timestamp` as well to be consistent though i guess that's a breaking change at this point

File: src/cozmo/objects.py [hunk 7]
```diff
@@ -283,6 +284,8 @@ def __init__(self, *a, **kw):
         self.last_tapped_time = None
 
         #: int: The robot's timestamp of the last tapped event
+        #: ``None`` if the cube wasn't tapped yet
+        #: In milliseconds relative to robot epoch
```
Comment (gwatts) [inline src/cozmo/objects.py]: Needs punctuation - Sphinx will run these lines together and it doesn't look good without it (plus i've been trying to be good about punctuation for Sphinx docs in general)

**Comment by MarkWesleyAtAnki** [comment]
@gwatts I've added full-stops after after all those lines
